letsencrypt-auto-source/pieces/bootstrappers/suse_common.sh

```diff
@@ -14,10 +14,18 @@ BootstrapSuseCommon() {
     QUIET_FLAG='-qq'
   fi
 
+  if zypper search -x python-virtualenv >/dev/null 2>&1; then
+    OPENSUSE_VIRTUALENV_PACKAGES="python-virtualenv"
+  else
+    # Since Leap 15.0 (and associated Tumbleweed version), python-virtualenv
+    # is a source package, and python2-virtualenv must be used instead.
+    OPENSUSE_VIRTUALENV_PACKAGES="python2-virtualenv python2-setuptools"
```

Comment:
nit: Can you add a comment about why we're including `python2-setuptools` so we remember in the future?

Reply:
Done.